index.js

```diff
@@ -118,55 +115,59 @@ function around(index, lng, lat, maxResults, maxDistance, predicate) {
 }
 
 // lower bound for distance from a location to points inside a bounding box
-function boxDist(lng, lat, node, cosLat, sinLat) {
+function boxDist(lng, lat, node) {
     var minLng = node.minLng;
     var maxLng = node.maxLng;
     var minLat = node.minLat;
     var maxLat = node.maxLat;
 
     // query point is between minimum and maximum longitudes
     if (lng >= minLng && lng <= maxLng) {
-        if (lat <= minLat) return earthCircumference * (minLat - lat) / 360; // south
-        if (lat >= maxLat) return earthCircumference * (lat - maxLat) / 360; // north
-        return 0; // inside the bbox
+        var dLat = (lat < minLat) ? (lat - minLat) : ((lat > maxLat) ? (lat - maxLat) : 0);
+        return haverSin(dLat * rad);
     }
 
     // query point is west or east of the bounding box;
     // calculate the extremum for great circle distance from query point to the closest longitude
-    var closestLng = (minLng - lng + 360) % 360 <= (lng - maxLng + 360) % 360 ? minLng : maxLng;
-    var cosLngDelta = Math.cos((closestLng - lng) * rad);
-    var extremumLat = Math.atan(sinLat / (cosLat * cosLngDelta)) / rad;
-
     // calculate distances to lower and higher bbox corners and extremum (if it's within this range);
     // one of the three distances will be the lower bound of great circle distance to bbox
-    var d = Math.max(
-        greatCircleDistPart(minLat, cosLat, sinLat, cosLngDelta),
-        greatCircleDistPart(maxLat, cosLat, sinLat, cosLngDelta));
-
+    var haverSinDLng = Math.min(haverSin((lng - minLng) * rad), haverSin((lng - maxLng) * rad));
```

Comment:
Would it be slightly faster to keep the `closestLng` line to above and avoid calculating `haverSin` twice here?